README.md

```diff
@@ -563,6 +564,32 @@ https://streamable.com/moo
 
 </details>
 
+### Testing Playground
+
+#### Usage
+
+```md
+https://testing-playground.com/gist/1750787c77ae724da475efd788230104/6d090f5501deedc70a97ff52260ccc22a6248414
+```
+
+<details>
+<summary><b>Result</b></summary>
+
+```html
+<iframe
+  src="https://testing-playground.com/embed/1750787c77ae724da475efd788230104/6d090f5501deedc70a97ff52260ccc22a6248414?panes=preview,result"
+  height="300"
```

Comment:
Maybe the height should be a little taller? This seems pretty short. Also, could it be configurable via query params?

Reply:
It's the default height you also get when pressing the `embed` button in the GUI.

We can add options for setting `width`/`height`, but those are not supported in this PR (yet)

Reply:
Should it not also call the `oembed` url instead of rewriting the url here? The oembed url supports alternative heights as per the spec:

Default
```
https://testing-playground.com/oembed?url=http%3A%2F%2Ftesting-playground.com%2Fembed%3F
```

```js
{
  ...
  "html": "<iframe src=\"http://testing-playground.com/embed?\" width=\"900\" height=\"300\" scrolling=\"no\" frameborder=\"0\" allowtransparency=\"true\" allowfullscreen=\"true\" title=\"Testing Playground\" style=\"overflow: hidden; display: block;\" loading=\"lazy\" name=\"testing-playground-1604009319947\"></iframe>",
  "width": 900,
  "height": 300,
  ...
}
```

Alternative height

```
https://testing-playground.com/oembed?url=http%3A%2F%2Ftesting-playground.com%2Fembed%3F&maxheight=450
```

```js
{
  ...
  "html": "<iframe src=\"http://testing-playground.com/embed?\" width=\"900\" height=\"450\" scrolling=\"no\" frameborder=\"0\" allowtransparency=\"true\" allowfullscreen=\"true\" title=\"Testing Playground\" style=\"overflow: hidden; display: block;\" loading=\"lazy\" name=\"testing-playground-1604009244659\"></iframe>",
  "width": 900,
  "height": "450",
  ...
}
```

Reply:
@smeijer It doesn't really matter tbh

Reply:
This is how it looks with `height=450`, I'm thinking about changing the default to that. What do you think?

Also, regarding the panes. We have 4, `markup`, `query`, `preview` and `result`. Which panes make the most sense to show as default? I'm in doubt between `markup + preview + query` and `markup + result + query`.

Reply:
I think `markup + preview + query` will always show the most in the embed, but having the `result` panel is like the most important one imo

Reply:
> @smeijer It doesn't really matter tbh

In that case, I would recommend using the api. That way, changes in playground won't break this plugin without me knowing so. The oembed api is something we can protect against breaking changes.

I've also added query params support for the `panes`, so those will also be configurable trough the oembed url. I'll push & deploy tomorrow.

Reply:
Lol, "this is how it looks", forgot to paste the codepen link :sweat_smile: 

https://codepen.io/smeijer/pen/yLYWZmK

Reply:
Update has been deployed, default height is now 450px, and can still be adjusted with the `maxheight` param. Thepanes are configurable with `panes=markup,query,preview,result` order of panes is being translated to pane order in the playground. So `panes=markup,preview` and `panes=preview,markup` result in different layouts.

Reply:
I decided to do it manually for now and if necessary (if people ask for it), I can add support for `height`/`width` options.

Going to work on a more durable/scalable solution for the future though.
I think either using the oEmbed endpoint instead of doing things manually **or** (like @smeijer pointed me out) looking at the `json+oembed` meta tag could be a better solution.
Although services like Slides, Spotify, Twitter (only for Statuses, not for Moments or Timelines) don't have that meta tag and Twitch even doesn't have an oEmbed endpoint.